src/clj/rems/api/public.clj

```diff
@@ -17,6 +17,7 @@
 
 (def GetConfigResponse
   {:authentication s/Keyword
+   (s/optional-key :login-options) s/Str
```

Comment:
Should this be named `:alternative-login-url` or something? I find that there is now login-options and alternative used as name. Plural also refers to there being multiple other options.

Reply:
good point, will change it